namespaces/api-namespace.md

```diff
@@ -42,6 +42,11 @@ For example a `category` element may be classified both as `resourceGroup` and
             - resourceGroup - Category is a set of resource.
             - dataStructures - Category is a set of data structures.
             - scenario - Reserved. Category is set of steps.
+- `attributes`
+    - `meta` (object) - Arbitrary metadata
```

Comment:
I think for the sake of clarity, this should be `metadata` here. It is something completely different than `meta` and the extra verbosity will keep the confusion down.